package.json

```diff
@@ -126,7 +126,7 @@
 		"istanbul": "1.1.0-alpha.1",
 		"istanbul-lib-coverage": "^1.2.0",
 		"istanbul-merge": "^1.1.1",
-		"jest": "^18 || ^19 || ^20.0.1 || ^21 || ^22 || ^23 || ^24 || ^25 || ^26",
+		"jest": "^27.2.0",
```

Comment:
the dev dep range must exactly match the peer dep range.

Reply:
Ok, I'll make this change.